docs/supported-languages.md

```diff
@@ -127,7 +89,7 @@ Additionally, Semgrep offers beta support for the scanning of Java projects **wi
    <td><code>packages.lock.json</code></td>
    <td style={{"text-align": "center"}}>GA</td>
    <td>✅</td>
-   <td rowspan="16">80% of all critical severity CVEs since 2017 and 100% of critical and high severity CVEs since May 2022</td>
+   <td rowspan="2">80% of all critical severity CVEs since 2017 and 100% of critical and high severity CVEs since May 2022</td>
```

Comment:
Why did we change this portion of the table? I don't think it quite works -- we can have it per language (too much, I think) or as a whole, but I think it's a bit visually overwhelming as it is

Reply:
See Slack for latest discussion.